syntax = "proto2";

package tensorflow.core.function.trace_type.default_types;

// Represents a serialized Literal type.
message SerializedLiteral {
  oneof value {
    bool bool_value = 1;
    int64 int_value = 2;
    double float_value = 3;  // Python "float" is double precision.
    string str_value = 4;
  }
}
